src/mame/video/starfield_05xx.cpp

```diff
@@ -0,0 +1,683 @@
+// license:BSD-3-Clause
+// copyright-holders:Robert Hildinger
+
+
+
+
+
+
```

Comment:
Please don't put big blank spaces like this.  One to three consecutive blank lines between sections, depending on the size of the sections, should be fine.

Reply:
Change committed